core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -249,20 +324,12 @@ func (ec *ethConfirmer) CheckForReceipts(ctx context.Context, blockNum int64) er
 			return errors.Wrap(err, "saveFetchedReceipts failed")
 		}
 	}
-
-	if err := ec.markConfirmedMissingReceipt(); err != nil {
-		return errors.Wrap(err, "unable to mark eth_txes as 'confirmed_missing_receipt'")
-	}
-
-	if err := ec.markOldTxesMissingReceiptAsErrored(blockNum); err != nil {
-		return errors.Wrap(err, "unable to confirm buried unconfirmed eth_txes")
-	}
-
 	return nil
 }
 
 func (ec *ethConfirmer) findEthTxAttemptsRequiringReceiptFetch() (attempts []models.EthTxAttempt, err error) {
 	err = ec.store.DB.
+		Joins("EthTx"). // Joins("EthTx") is needed for the query to actually return data from eth_txes table as well.
```

Comment:
Oooh does this autopreload it? That's cool!

Reply:
yes - was a bit surprising that it's not the default.. :)